lib/extensions/src/lib/services/extension-loader.service.ts

```diff
@@ -166,4 +166,12 @@ export class ExtensionLoaderService {
         }
         return action;
     }
+
+    private filterIgnoredExtensions(extensions: string[], config: ExtensionConfig): string[] {
```

Comment:
Why passing down the whole config object? I assume the $ignoreReferenceList would be enough, this is the only thing what we use from this object, as I see.

Reply:
ah, yes